docs/backend/roles.md

```diff
@@ -33,21 +33,21 @@ at the console.
 rails console
 ```
 
-- after verifying the user `test_user_name` is missing the `pro` role we proceed
-  to add it and then verify the role has been added:
+- after verifying the user `test_user_name` is missing the `trusted` role we
```

Comment:
This just updates the docs so that the example we use for "assigning user roles" isn't an example that wouldn't actually work in the console for someone locally! 😛